### PR TITLE
Added some missing names for HudElements functions

### PR DESCRIPTION
**Pull request by eldexterr**

### CODE REVIEW

File: include/functions.h [hunk 1]
```diff
@@ -79,7 +79,7 @@ void func_80258E14(void);
 f32 func_800E34D8(void);
 f32 player_check_collision_below(f32, s32* colliderID);
 s32 can_trigger_loading_zone(void);
-void func_802667F0(s32, Actor*, f32, f32, f32);
+void func_create_textPopup(s32, Actor*, f32, f32, f32);
```
Comment (nanaian) [inline include/functions.h:82]: You don't need the "func" prefix. what does this function actually do? the mix of snake and camel case is a bit weird to me

**Review by nanaian — COMMENTED** [review]
Looks like there are some nasty merge conflicts that need to be resolved to merge this

File: include/functions.h (hunk 1 above)
Comment (eldexterr) [reply include/functions.h]: Oh my bad, I already renamed the function 👍 , it creates the little Nice, Good, Super text that appears when you land action commands successfully.

**Comment by eldexterr** [comment]
Hmm well I dunno what I did to fuck all those files lol, I will just close this pr for now

**Pull request closed without merging**